spec.html

```diff
@@ -14104,7 +14104,7 @@ <h1>Runtime Semantics: HostGetImportMetaProperties ( _moduleRecord_ )</h1>
           <h1>Runtime Semantics: HostFinalizeImportMeta ( _importMeta_, _moduleRecord_ )</h1>
           <p>HostFinalizeImportMeta is an implementation-defined abstract operation that allows hosts to perform any extraordinary operations to prepare the object returned from `import.meta`.</p>
 
-          <p>Most hosts will be able to simply define HostGetImportMetaProperties, and leave HostFinalizeImportMeta with its default behavior. However, HostFinalizeImportMeta provides an "escape hatch" for hosts which need to directly manipulate the object before it is exposed to ECMAScript code.</p>
+          <p>Most hosts will be able to simply define HostGetImportMetaProperties, and leave HostFinalizeImportMeta with its default behaviour. However, HostFinalizeImportMeta provides an "escape hatch" for hosts which need to directly manipulate the object before it is exposed to ECMAScript code.</p>
```

Comment:
this gives me a sad demeanour

Reply:
https://en.wikipedia.org/wiki/Oxford_spelling

> It is common for academic, formal, and technical writing for an international readership

Reply:
@ljharb I appreciate your candour.